src/components/BottomSheet/BottomSheetMenu.js

```diff
@@ -40,7 +40,7 @@ const styles = StyleSheet.create({
     height: dimen.normalizeDimen(height)
   }),
   rbsheetDraggableIcon: {
-    backgroundColor: COLORS.greySubtile1,
+    backgroundColor: COLORS.gray200,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in color from `COLORS.greySubtile1` to `COLORS.gray200` seems fine as long as the new color is consistent with the overall theme and design of the application. Ensure that this color change doesn't affect the readability or user experience.

```diff
-    backgroundColor: COLORS.greySubtile1,
+    backgroundColor: COLORS.gray200,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->